scripts/fit_image.py

```diff
@@ -0,0 +1,71 @@
+"""
+A python tool to resize images 
+Usage:
+    python fit_image.py --images xxxx 
+if you wanna set image resolution ,please use this command :
+    python fit_image.py --images xxxx --width 1920 --height 1080  or
+    python fit_image.py --images xxxx --width 1920 or 
+    python fit_image.py --images xxxx  --height 1080 
+if you wanna set image resolution by ratio ,please use this command:
+    python fit_image.py --images xxx --ratio 0.5
+
+"""
+
+import os
+from PIL import Image
+import argparse
+import re
+
+
+def resize_image(image_path, args):
+    img = Image.open(image_path)
+    w, h = img.size
+    if args.ratio > 0:
+        w_hat = int(w * args.ratio)
+        h_hat = int(h * args.ratio)
+    else:
+        if w > args.width > 0 and h > args.height > 0:

```

Comment:
This code style is a little hard to read. It would be better if we can use some more meaningful code instead of `w > args.width > 0`.

Reply:
OK,I will .